easybuild/easyconfigs/s/Shapely/Shapely-2.0.1-foss-2023a.eb

```
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Updated: Denis Kristak
+easyblock = 'PythonPackage'
+
+name = 'Shapely'
+version = '2.0.1'
+
+homepage = 'https://github.com/Toblerity/Shapely'
+description = """Shapely is a BSD-licensed Python package for manipulation and analysis of planar geometric objects.
+It is based on the widely deployed GEOS (the engine of PostGIS) and JTS (from which GEOS is ported) libraries."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Since this EasyConfig does not require MPI, I think we could move this in the module hierarchy to the `gfbf` toolchain (requires also a rename the file to `Shapely-2.0.1-gfbf-2023a.eb`):

```suggestion
toolchain = {'name': 'gfbf', 'version': '2023a'}
```